lib/defaults.js

```diff
@@ -30,6 +30,24 @@ var defaults = module.exports = {
   //from the pool and destroyed
   poolIdleTimeout: 30000,
 
+  // block sets the behavior of node-postgres when the connection pool is
+  // full. Set to false to immediately return an error if no connections are
+  // available. By default, node-postgres will wait an infinite amount of time
+  // for a connection to become available.
+  block: true,
+
+  // acquireTimeout is the amount of time in ms to wait to check out a
+  // database connection. By default, node-postgres will wait infinitely for
+  // a connection to become available. If the acquireTimeout is negative, zero,
+  // or NaN, pg.connect will return an error immediately.
+  //
+  // If block is set to false, this setting is ignored - node-postgres will
+  // error immediately if block is false and the pool is full.
+  //
+  // The timeout message will be "Cannot acquire resource because the pool is
+  // full"
+  acquireTimeout: undefined,
```

Comment:
Related to `block = true` above - this _could_ be a single property - if you set `acquireTimeout === 0` then that could be the same meaning as `block = true` removing the need for the `block` property. 

So, 
```js

acquireTimeout: 0 //timeout immediately, effectively block: true
acquireTimeout: > 0 //timeout after this amount of time
acquireTimeout: <any other value> //noop - same behavior as existing which is to wait 'forever' to timeout
```

Up to you on the implementation & decision here 100% - just putting in my 2 cents.  Less properties is nice because then you don't have to explain how a `block = false && acquireTimeout = 100` will interplay with one another; however, explicitness can be a nice feature of an API as well.